src/dfs/dfs_velocity.cpp

```diff
@@ -147,23 +148,22 @@ namespace SpherePoisson {
             Int nrows = vort.extent(0);
             Int ncols = vort.extent(1);
             view_2d<Complex> tmp("temp", nrows, ncols);
+            view_2d<Complex> tmp_th("temp", nrows, ncols);
```

Comment:
Same thing here: the views must be declared outside of the kernel, and then the `KOKKOS_LAMBDA` capture will copy them in.

Reply:
It looks like you need two 2d arrays of complex values for this kernel; these views must be declared (and their memory allocated) outside of the `parallel_for` call, not inside.